public/css/app.css

```diff
@@ -474,50 +477,40 @@ form span {
   right: 0;
 }
 
-.input-group-button .loader,
-.input-group-button.loading-data .loader {
-  transition: all 0.25s ease-out;
+.form-group.loading-data .button,
+.form-group.loading-data input::placeholder,
+.form-group.loading-data input {
+  color: rgba(0, 0, 0, 0) !important;
 }
 
-.input-group-button.loading-data .loader {
+.form-group.loading-data .loader {
   display: block;
   opacity: 0.58;
   width: 25px;
   height: auto;
   visibility: visible;
 }
 
-.input-group-button.loading-data .button {
-  color: rgba(255, 255, 255, 0);
+.form-group.loading-data .input-group-button,
+.form-group.loading-data .button {
+  color: rgba(255, 255, 255, 0) !important;
```

Comment:
Actually, I'm a bit confused here, now that I read this again. Above on L480, we define the `.form-group.loading-data .button {}` with a color of `rgba(#000, 0) !important` but here we're overriding the same selector with a value of `rgba(#fff, 0) !important`. Since the alpha values are 0 in both cases, not sure it matters, but we could possibly drop `.form-group.loading-data .button` from one group or the other.

```diff
+.form-group.loading-data .button, ... {
+  color: rgba(0, 0, 0, 0) !important;
 }
```